source/fundamentals/connection/connection-options.txt

```diff
@@ -0,0 +1,225 @@
+
+   * - **heartbeatFrequencyMS**
+     - integer greater than or equal to 500
+     - ``10000``
+     - The interval between regular server-monitoring checks.
+
+   * - **journal**
+     - Boolean
+     - ``null``
+     - Specifies the default write concern ``"j"`` field for the client. See
+       :ref:`write concern <wc-j>` for more information.
+
+   * - **loadBalanced**
+     - Boolean
+     - ``false``
+     - Specifies whether the driver is connecting to a load balancer.
+
+   * - **localThresholdMS**
+     - nonnegative integer
+     - ``15``
+     - The latency window for server eligibility. If a server's round trip takes longer than the fastest server's round-trip time plus this value, the server isn't eligible for selection. Specifying ``0`` selects the fastest eligible server.
+
+   * - **maxIdleTimeMS**
+     - nonnegative integer
+     - ``10000``
+     - The length of time a connection can be idle before the driver closes it.
+       Specifying ``0`` means no minimum.
+
+   * - **maxPoolSize**
+     - nonnegative integer
+     - ``100``
+     - The greatest number of clients or connections the driver
+       can create in its connection pool. This count includes connections
+       in use.
+
+   * - **maxConnecting**
+     - nonnegative integer
+     - ``2``
+     - The greatest number of connections a driver's connection
+       pool may be establishing concurrently.
+
+   * - **maxStalenessSeconds**
+     - -1, or an integer greater than or equal 90
+     - ``-1``
+     - The longest replication lag, in wall-clock time, that
+       a secondary can experience and still be eligible for server selection.
+       Specifying ``-1`` means no maximum.
+
+   * - **minPoolSize**
+     - nonnegative integer
+     - ``0``
+     - The number of connections the driver should create and
+       keep in the connection pool even when no operations are occurring.
+       This count includes connections in use.
+
+   * - **readConcernLevel**
+     - string
+     - ``null``
+     - The client's default read concern. See :ref:`read concern
+       <read-concern>` for more information.
+
+   * - **readPreference**
+     - string
+     - ``"primary"``
+     - The client's default read preference (excluding tags). See :ref:`read preference <read-preference>` for more information.
+
+   * - **readPreferenceTags**
+     - comma-separated ``key:value`` pairs (for example, ``opt1:val1,opt2:val2``)
+     - ``null``
+     - The default read-preference tags for the client. This option is
+       valid only if the read-preference mode is not ``primary``.
+       
+       You can include this parameter in the connection URI more than once. If you do, the client treats each instance as a separate tag set. The order of the tags in the URI determines the order for read preference.
+
+   * - **replicaSet**
+     - string
+     - ``null``
+     - The name of the replica set to connect to.
+
+   * - **retryReads**
+     - Boolean
```

Comment:
issue: Isn't the keyword `bool` in C#, not `Boolean`? Otherwise, I believe we should capitalize all types for their .NET type.

Reply:
Yeah, it's `bool` in C# code but "Boolean" in text. This relates back to the previous comment about the importance of using C# data types in this context.

Reply:
For this I think it should be `bool`,  otherwise, I think we have to change other instances to `String`.

Reply:
I followed the pattern in the spec for leaving them uncapitalized, but made it "Boolean" because that's technically correct. I'd prefer capitalizing every data type in this column, personally

Reply:
Then let's capitalize them. The spec is a good starting point but it's on us to make it idiomatic for the language.